docs/en/writing-running-appium/tutorial/swipe/ios-mobile-element.md

```diff
@@ -0,0 +1,103 @@
+## iOS 'mobile:': Element swipe
+
+To swipe inside element you need add element id into swipe methods.
```

Comment:
...inside of an element you need to add the id of this element into swipe methods